Tests/IntegrationTests.Shared/RealmResults/SortingTests.cs

```diff
@@ -223,6 +223,25 @@ public void FirstIsDifferentSorted()
             Assert.That(sortedFirst.Email, Is.EqualTo("john@doe.com"));
         }
 
+
+        [Test]
+        public void LastIsDifferentSorted()
```

Comment:
Then perhaps we could look for a more suitable name - e.g. `LastWorksWithOrderBy`? Also, since we have such test for `.Last`, we should probably do it for `.First`, since it can also be different based on sorting.